hack/non-olm-install/install-gitops-operator.sh

```diff
@@ -0,0 +1,254 @@
+kind: Kustomization
+namespace: ${NAMESPACE_PREFIX}system
+namePrefix: ${NAMESPACE_PREFIX}
+resources:
+  - https://github.com/redhat-developer/gitops-operator/config/crd?ref=$GIT_REVISION
+  - https://github.com/redhat-developer/gitops-operator/config/rbac?ref=$GIT_REVISION
+  - https://github.com/redhat-developer/gitops-operator/config/manager?ref=$GIT_REVISION
+patches:
+  - path: https://raw.githubusercontent.com/redhat-developer/gitops-operator/master/config/default/manager_auth_proxy_patch.yaml 
+  - path: env-overrides.yaml" > ${TEMP_DIR}/kustomization.yaml
+}
+
+# creates a patch file, containing the environment variable overrides for overriding the default images
+# for various gitops-operator components.
+function create_image_overrides_patch_file() {
+  echo "[INFO] Creating env-overrides.yaml file using component images specified in environment variables"
+  echo "apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+      - name: manager
+        image: ${OPERATOR_IMG}
+        env:
+        - name: ARGOCD_DEX_IMAGE
+          value: ${ARGOCD_DEX_IMAGE}
+        - name: ARGOCD_KEYCLOAK_IMAGE
+          value: ${ARGOCD_KEYCLOAK_IMAGE}
+        - name: ARGOCD_APPLICATIONSET_IMAGE 
+          value: ${ARGOCD_APPLICATIONSET_IMAGE}
+        - name: BACKEND_IMAGE
+          value: ${BACKEND_IMAGE}
+        - name: ARGOCD_IMAGE
+          value: ${ARGOCD_IMAGE}
+        - name: ARGOCD_REPOSERVER_IMAGE
+          value: ${ARGOCD_IMAGE}
+        - name: ARGOCD_REDIS_IMAGE
+          value: ${ARGOCD_REDIS_IMAGE}
+        - name: ARGOCD_REDIS_HA_IMAGE
+          value: ${ARGOCD_REDIS_IMAGE}
+        - name: ARGOCD_REDIS_HA_PROXY_IMAGE
+          value: ${ARGOCD_REDIS_HA_PROXY_IMAGE}
+        - name: GITOPS_CONSOLE_PLUGIN_IMAGE
+          value: ${GITOPS_CONSOLE_PLUGIN_IMAGE}
+        - name: KAM_IMAGE
+          value: ${KAM_IMAGE}" > ${TEMP_DIR}/env-overrides.yaml
+}
+
+function create_deployment_patch_from_bundle_image() {
+  echo "[INFO] Creating env-overrides.yaml file using component images specified in the gitops operator bundle image"
+  container_id=$(${DOCKER} create --entrypoint sh "${BUNDLE_IMG}")
+  ${DOCKER} cp "$container_id:manifests/gitops-operator.clusterserviceversion.yaml" "${TEMP_DIR}"
+  ${DOCKER} rm "$container_id"
+
+  echo "apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system" > "${TEMP_DIR}"/env-overrides.yaml
+  cat "${TEMP_DIR}"/gitops-operator.clusterserviceversion.yaml | ${YQ} -e '.spec.install.spec.deployments[0]' | tail -n +2 >> "${TEMP_DIR}"/env-overrides.yaml
+  ${YQ} -e -i '.spec.selector.matchLabels.control-plane = "argocd-operator"' "${TEMP_DIR}"/env-overrides.yaml
+  ${YQ} -e -i '.spec.template.metadata.labels.control-plane = "argocd-operator"' "${TEMP_DIR}"/env-overrides.yaml
+  cat "${TEMP_DIR}"/env-overrides.yaml
+}
```

Comment:
@anandf on trying to run this script I see the following warning 

```Warning: would violate PodSecurity "restricted:v1.24": seccompProfile (pod or containers "manager", "kube-rbac-proxy" must set securityContext.seccompProfile.type to "RuntimeDefault" or "Localhost")```

maybe we should add the appropriate security context  to the operator deployment so it can run on k8s > 1.24

Reply:
yes. I will look into this. Probably needs fix in the manifests. Wondering how it works with OLM based installation. Does it handle it using Openshift Security Context Constraints (SCC)?

Reply:
I have updated the Security Context for the containers within the pod in my latest commit. There are no `Warnings` now when running this script.